src/app/loadout/plug-drawer/PlugDrawer.tsx

```diff
@@ -12,22 +12,11 @@ import React, { RefObject, useCallback, useEffect, useMemo, useRef, useState } f
 import Sheet from '../../dim-ui/Sheet';
 import '../../item-picker/ItemPicker.scss';
 import Footer from './Footer';
-import PlugSection from './PlugSection';
-
-export interface PlugsWithMaxSelectable {
-  plugSetHash: number;
-  plugs: PluggableInventoryItemDefinition[];
-  maxSelectable: number;
-}
+import PlugSection, { PlugsWithMaxSelectable } from './PlugSection';
 
 interface Props {
-  /**
-   * A list of plug items that come from a PlugSet, along with the maximum number of these plugs that can be chosen.
-   */
   plugsWithMaxSelectableSets: PlugsWithMaxSelectable[];
```

Comment:
Comment isn't useful?

Reply:
felt redundant with the more detailed one attached to the `PlugsWithMaxSelectable` type